### DISCUSSION TITLE
Full home directory path shown instead of ~

### DISCUSSION THREAD

**Discussion by jpoppe** [marlonrichert/zsh-snap#274] [Q&A]
Since commit  https://github.com/marlonrichert/zsh-snap/commit/af1130996eb397ee9530940c4df38d27c796ab87 my pure prompt shows `/home/jpoppe` instead of `~`.

I suspect this has to do with the `Named dirs`  ([~..]) implementation.

Anyone knows how to configure `zsh` to show the  tilde (~) instead of the full home path?

**Comment by marlonrichert** [comment]
It works correctly for me. What prompt do you use?
